### PR TITLE
[WIP] #3476 : WebAssembly bulk operations implementation

### PR DESCRIPTION
**Pull request by Fly-Style**

### CODE REVIEW

File: lib/WasmReader/WasmBinaryOpCodes.h [hunk 1]
```diff
@@ -66,6 +66,7 @@
 
 WASM_PREFIX(Numeric, WASM_PREFIX_NUMERIC, Wasm::WasmNontrapping::IsEnabled(), "WebAssembly nontrapping float-to-int conversion support is not enabled")
 WASM_PREFIX(Threads, WASM_PREFIX_THREADS, Wasm::Threads::IsEnabled(), "WebAssembly Threads support is not enabled")
+WASM_PREFIX(Bulk, WASM_PREFIX_BULK, Wasm::Threads::IsEnabled(), "WebAssembly Threads support is not enabled")
```
Comment (MikeHolman) [inline lib/WasmReader/WasmBinaryOpCodes.h]: should be Wasm::Bulk::IsEnabled

File: lib/WasmReader/WasmBinaryOpCodes.h [hunk 2]
```diff
@@ -170,7 +171,12 @@ WASM_MEMSTORE_OPCODE(I64StoreMem16, 0x3d, L_IL, true, Js::ArrayBufferView::TYPE_
 WASM_MEMSTORE_OPCODE(I64StoreMem32, 0x3e, L_IL, true, Js::ArrayBufferView::TYPE_INT32_TO_INT64, "i64.store32")
 
 WASM_MISC_OPCODE(MemorySize, 0x3f, I_I, true, "memory.size")
-WASM_MISC_OPCODE(MemoryGrow,    0x40, I_I, true, "memory.grow")
+WASM_MISC_OPCODE(MemoryGrow, 0x40, I_I, true, "memory.grow")
+#ifdef ENABLE_WASM_BULK
+WASM_MISC_OPCODE(MemoryCopy, 0xd1, I_I, true, "memory.copy")
+WASM_MISC_OPCODE(MemoryFill, 0xd2, I_I, true, "memory.fill")
```
Comment (MikeHolman) [inline lib/WasmReader/WasmBinaryOpCodes.h]: instead of true, you want `Wasm::Bulk::IsEnabled()`

Comment (MikeHolman) [inline lib/WasmReader/WasmBinaryOpCodes.h]: instead of `I_I`, we should make a new `WASM_SIGNATURE` for void return and I32, I32, I32 parameters

Comment (Fly-Style) [reply lib/WasmReader/WasmBinaryOpCodes.h]: Thanks for the review! 👍

**Pull request closed without merging**